libs/langchain/langchain/graphs/falkordb_graph.py

```diff
@@ -52,9 +52,9 @@ def get_schema(self) -> str:
     def refresh_schema(self) -> None:
         """Refreshes the schema of the FalkorDB database"""
         self.schema = (
-            f"Node properties: {node_properties_query}\n"
-            f"Relationships properties: {rel_properties_query}\n"
-            f"Relationships: {rel_query}\n"
+            f"Node properties: {self.query(node_properties_query)}\n"
```

Comment:
nit: we're following google style guide for python, the queries which are global constants should be capitalized: https://google.github.io/styleguide/pyguide.html#316-naming